src/registry/index.js

```diff
@@ -0,0 +1,20 @@
+import debugFactory from 'debug';
+import isApiValid from '../utils/is-api-valid';
+
+const debug = debugFactory( 'fresh-data:registry' );
+const _apis = new Map();
+
+export function registerApi( api, apis = _apis ) {
+	debug( `Registering api "${ api && api.name }"` );
+
+	if ( ! isApiValid( api ) ) {
+		throw new Error( 'Invalid API object' );
```

Comment:
Should the error message be a bit more descriptive about _why_ the API object was invalid? Just thinking that might make it easier for 3pd to decipher what went wrong.

Reply:
I see it is output as a debug, disregard.